sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.institutions.controller.ts

```diff
@@ -311,19 +307,6 @@ export class ConfirmationOfEnrollmentInstitutionsController extends BaseControll
       disbursementSchedule.studentAssessment.application.applicationStatus,
       payload.tuitionRemittanceAmount,
     );
-
```

Comment:
Are we not sending COE confirmation to workflow anymore(Did not see the update status to enrollment in workflow as well)? if so why?

Reply:
**Previous workflow**
![image](https://user-images.githubusercontent.com/61259237/196819889-0e5f6d26-bd3d-445e-b1f5-53950bcbe4dd.png)

**Current workflow**
![image](https://user-images.githubusercontent.com/61259237/196819939-e6bc60d9-e298-4c17-8517-cf074506c1eb.png)

The only change from the previous to the current is that the "wait for the institution to complete the COE" is no longer there because right now there is nothing to happen after that message is received.
It was removed as per @abschwenker suggestion and it can easily be added back if needed.
Please let me know if it is clear or if we need further discussions.

Reply:
I get it. From a workflow perspective there is no significance of workflow waiting for COE to be confirmed. 👍